<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шифр Виженера</string>
  </property>
  <widget class="QRadioButton" name="uncode_radioButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Дешифровать</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="code_radioButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Шифровать</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Операция:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>751</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ввод</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeword_lineEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_data_textBrowser">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>378</y>
     <width>751</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>ГЕНЕРИРОВАТЬ</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_data_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>751</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Код-фраза:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Вывод</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Шифр Виженера</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
